src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  filter: (filter: Filter) => void
+  removeCompletedTodos: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  removeCompletedTodos: removeAll,
+}) => {
+  const [activeFilter, setActiveFilter] = useState({
+    all: true,
+    active: false,
+    completed: false,
+  });
```

Comment:
This state is redundant. You don't need it at all.
You can pass `currentFilter ` prop to the `Footer` component from `App` component to get needed classes (see below)